recipes/ledger/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "ledger" %}
+{% set version = "3.1.3" %}
+{% set sha256 = "b248c91d65c7a101b9d6226025f2b4bf3dabe94c0c49ab6d51ce84a22a39622b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ledger/ledger/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - fix-build.patch
+
+build:
+  number: 0
+  skip: True  # [win or osx]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - python >=2.4,<3
+  host:
+    - boost >=1.49
```

Comment:
You need to remove the versions here for conda-build to use the global values from `conda_build_config.yaml`.